src/Components/CustomElements/src/js/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "private": true,
   "scripts": {
-    "preclean": "yarn install --mutex network --frozen-lockfile",
+    "preclean": "yarn install --mutex network --frozen-lockfile || yarn install --mutex network --frozen-lockfile",
```

Comment:
This may not work (but we should try anyway). Depending on how this is ultimately invoked in CI/PR builds, writing to STDERR might actually be enough to mark the build as failed.